### PR TITLE
Multi arch support for k8s-lifecycle-metrics

### PR DESCRIPTION
**Pull request by myaser**
teapot/issues#3283

### CODE REVIEW

File: cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: kubernetes-lifecycle-metrics
       containers:
         - name: kubernetes-lifecycle-metrics
-          image: "pierone.stups.zalan.do/teapot/kubernetes-lifecycle-metrics:master-9"
+          image: "926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/kubernetes-lifecycle-metrics:master-12"
```
Comment (mikkeloscar) [inline cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml]: We should be able to use `container-registry.zalando.net` URL here as it goes through the admission-controller.

Comment (myaser) [reply cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml]: 👍🏻

**Comment by myaser** [comment]
👍🏻

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**